modules/bibupload/lib/bibupload.py

```diff
@@ -288,6 +289,7 @@ def bibupload(record, opt_mode=None, opt_notimechange=0, oai_rec_id="", pretend=
                 error = None
         else:
             write_message("   Note: 005 already existing upon inserting of new record. Keeping it.", verbose=2)
+        original_005 = time.strftime("%Y-%m-%d %H:%M:%S", time.strptime(record['005'][0][3].split('.')[0], "%Y%m%d%H%M%S"))
```

Comment:
Then it would be wrong, The meaning of `original_005` is in terms of the flow of code, not WRT where 005 comes from. An `original_005` value has always to be initilaized in `insert_mode_p == True`